class/frontend-form-post.php

```diff
@@ -182,6 +183,8 @@ function submit_post() {
         $is_update           = false;
         $post_author         = null;
         $default_post_author = wpuf_get_option( 'default_post_owner', 'wpuf_general', 1 );
+        $guest_verify = $form_settings['guest_email_verify'];
```

Comment:
Check with `isset()` to remove PHP warnings. Older users will not have this array and will throw PHP warnings.